packages/core/src/odata-common/request-builder/batch/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './batch-change-set';
```

Comment:
Now we export more files, because it's necessary after slitting to multiple packages in v2.0.
I guess we can detect the potential name conflicts (e.g. same class names in common/v2/v4 folders) via tests.

Reply:
for me the compiler already complains if there are conflicting exports.